tests/testsuite/build.rs

```diff
@@ -4213,6 +4217,7 @@ fn message_format_json_forward_stderr() {
             "overflow_checks": false,
             "test":false
         },
+        "executable": "{...}",
```

Comment:
Not verifying the executable here since that's not what's being tested here. Is that OK?

Reply:
Yep, that's ok!